sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -226,6 +248,7 @@ public BlobBaseClient(string connectionString, string blobContainerName, string
             _customerProvidedKey = options.CustomerProvidedKey;
             _clientSideEncryption = options._clientSideEncryptionOptions?.Clone();
             _encryptionScope = options.EncryptionScope;
+            _storageSharedKeyCredential = StorageSharedKeyCredential.ParseConnectionString(connectionString);
```

Comment:
Can we just use the `conn` variable instead of parsing the connection string a second time?  [`conn.Credentials as StorageSharedKeyCredential;`](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Common/src/Shared/StorageConnectionString.cs#L223) will get you what you need here.